source/spark_sql_migrations/setup.py

```diff
@@ -10,8 +10,8 @@
     license="MIT",
     packages=find_packages(),
     install_requires=[
-        "pyspark==3.3.0",
-        "delta-spark==2.3.0",
+        "pyspark==3.5.*",
```

Comment:
Why wildcard `pyspark` minors, and not e.g., `delta-spark` and `dependency_injector`?

Reply:
I will set the Pyspark to 3.5.1 because that is the newest version. 

I like the specific versions way better and no good reason for doing this :P